Code/C++/Brian_Kernighan.cpp

```diff
@@ -49,9 +47,9 @@ int count_set(int n){
 int main(){
     int n;
     // Taking input from the user
-    cin>>n;   
+    cin >> n;   
     int ans = count_set(n);
-    cout<<ans<<"\n";
+    cout << ans << "\n";
 }
 
 /* TEST CASES
```

Comment:
use lowercase as `Test case :`

Reply:
I have made the changes that were asked. Please review it and let me know if any other changes are required. I am open to learning. If it is okay I would request you to merge this.  
Regards,
Debanjan Poddar
@rudrakshi99 
@tarun26091999 
@plazzy99 
@raksha009